Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -100,9 +117,18 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Progression System Spot Component"))
 	TObjectPtr<class UPSSpotComponent> PSCurrentSpotComponentInternal = nullptr;
 
-	/** Store the current save game instance */
+	/** Store the current save game instance
+	 * Contains the FPSSaveToDiskData which has actual data from save file */
+	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Save Game Data Internal"))
+	TObjectPtr<class UPSSaveGameData> SaveGameDataInternal = nullptr;
+
+	/** Store the Progression Settings data instance */
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Save Game Instance"))
-	TObjectPtr<class UPSSaveGameData> SaveGameInstanceInternal = nullptr;
+	TMap<FName, FPSRowData> ProgressionSettingsDataInternal;
+
+	/** Store the current row name */
+	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Current Row Name"))
+	FName CurrentRowNameInternal;
```

Comment:
- add initialization `= NAME_None`
- add `Transient` specifier for this runtime property

Expected result:
```
	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Current Row Name"))
	FName CurrentRowNameInternal = NAME_None;
Comment
```

Reply:
added: https://github.com/h4rdmol/ProgressionSystem/pull/18/commits/f2f7edd58c3ec87c0390454e4cf04571f36e5d2d